src/utils/analytics.ts

```diff
@@ -105,6 +105,7 @@ export const transformCycle = async (cycle: Cycle) => {
             values.push(item?.appJoinData?.stakeCert?.nominator ?? null)
             values.push(item.nodeInfo.address)
             values.push(cycleRecordStartISOString)
+            values.push(item.version)
```

Comment:
actually, item.version is not the shardeum version. we would have to get the shardeumVersion from appJoinData instead